provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -555,6 +556,26 @@ def get_code_from_file(file_path, store_type=None, credentials=None):
     return code
 
 
+def get_s3_object(file_path, credentials):
+    aws_region = credentials.get("aws_region")
+    aws_access_key_id = credentials.get("aws_access_key_id")
+    aws_secret_access_key = credentials.get("aws_secret_access_key")
+    bucket_name = credentials.get("aws_bucket_name")
+    if not (aws_region and aws_access_key_id and aws_secret_access_key and bucket_name):
+        raise Exception(
+            "the values for 'aws_region', 'aws_access_key_id', 'aws_secret_access_key', 'aws_bucket_name' need to be set as credential"
```

Comment:
super nit: the exception reads a bit funny to me. or instead "Missing credential values for 'x', 'y', 'z',...".